### PR TITLE
Issue 285: Parse About page from README.md

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
This pull request reads in the README.md file, uses Pandoc to convert the content from Markdown to Blaze HTML and adds the result to AboutResponse.hs.

Fixes: #285

### CODE REVIEW

File: hs/AboutResponse.hs [hunk 1]
```diff
@@ -18,63 +19,9 @@ aboutResponse =
...
-    "pointing out bugs or incorrect information, "
-    "and general thoughts on the design of our tools. "
-    H.strong "You don't need to be a programmer to get involved."
-  H.p $ do
-    "We can be reached at "
-    H.a ! A.href "mailto:courseography@cs.toronto.edu" ! A.target "_blank"
-        $ "courseography@cs.toronto.edu"
-    ", and look forward to hearing from you!"
+aboutHtml :: String -> H.Html
+aboutHtml contents = H.div ! A.id "aboutDiv" $ writeHtml def $ readMarkdown def $ contents
```
Comment (david-yz-liu) [inline hs/AboutResponse.hs]: The final $ isn't necessary.

Comment (Ian-Stewart-Binks) [inline hs/AboutResponse.hs]: Yeap. Fixed.

File: hs/AboutResponse.hs [hunk 2]
```diff
@@ -18,63 +19,11 @@ aboutResponse =
...
-    H.strong "You don't need to be a programmer to get involved."
-  H.p $ do
-    "We can be reached at "
-    H.a ! A.href "mailto:courseography@cs.toronto.edu" ! A.target "_blank"
-        $ "courseography@cs.toronto.edu"
-    ", and look forward to hearing from you!"
+-- | AboutHtml takes in the contents of the README.md file (the GitHub README file) and translates
+-- the markdown to blaze-HTML.
+aboutHtml :: String -> H.Html
+aboutHtml contents = H.div ! A.id "aboutDiv" $ writeHtml def $ readMarkdown def contents
```
Comment (david-yz-liu) [inline hs/AboutResponse.hs]: I think I'd prefer it if this were just a generic function to transform markdown into html, and you used the wrapper "aboutDiv" above. Down the road, we may have lots of markdown files converted.

Comment (Ian-Stewart-Binks) [inline hs/AboutResponse.hs]: Makes sense, good idea. Should I keep that function in the AboutResponse.hs file, or should I create a new... utility file?

Comment (david-yz-liu) [inline hs/AboutResponse.hs]: Eh, you can keep it here for now.

Comment (Ian-Stewart-Binks) [inline hs/AboutResponse.hs]: Done.

**Comment by christinem** [comment]
Just a note that maybe should be made more explicit: this introduces a new Haskell package that needs to be installed through cabal in order to be able to run the server again (pandoc). I realized this when my server refused to run and kept giving me an error about missing module Text.Pandoc. Incase anyone else runs into this issue.

**Comment by Ian-Stewart-Binks** [comment]
Good note Christine, thanks!

**Pull request merged**